test/data/student-finance-forms-responses-and-expected-results.yml

```diff
@@ -337,42 +337,42 @@
 - :current_node: :what_year_full_time?
   :responses:
   - eu-full-time
-  - year-1516
+  - year-1617
   :next_node: :continuing_student?
   :outcome_node: false
 - :current_node: :continuing_student?
   :responses:
   - eu-full-time
-  - year-1516
+  - year-1617
   - continuing-student
-  :next_node: :outcome_eu_ft_1516_continuing
+  :next_node: :outcome_eu_ft_1617_continuing
   :outcome_node: true
 - :current_node: :continuing_student?
   :responses:
   - eu-full-time
-  - year-1516
+  - year-1617
   - new-student
-  :next_node: :outcome_eu_ft_1516_new
+  :next_node: :outcome_eu_ft_1617_new
   :outcome_node: true
 - :current_node: :what_year_full_time?
   :responses:
   - eu-full-time
-  - year-1617
+  - year-1516
   :next_node: :continuing_student?
   :outcome_node: false
 - :current_node: :continuing_student?
   :responses:
   - eu-full-time
-  - year-1617
+  - year-1516
   - continuing-student
-  :next_node: :outcome_eu_ft_1617_continuing
+  :next_node: :outcome_eu_ft_1516_continuing
   :outcome_node: true
 - :current_node: :continuing_student?
   :responses:
   - eu-full-time
-  - year-1617
+  - year-1516
   - new-student
-  :next_node: :outcome_eu_ft_1617_new
+  :next_node: :outcome_eu_ft_1516_new
   :outcome_node: true
 - :current_node: :type_of_student?
   :responses:
```

Comment:
It looks as if all these changes to responses & expected results should have been included in this commit: abd3fe1a29b67d134a7f0d0de4f73135c21ef5b2 which was part of #2296. It's odd that both the questions & responses and responses & expected results files were updated in that commit, but the latter did not match the former.

Anyway now that I understand a bit more about what happened, I'm happy to include these changes.